### DISCUSSION TITLE
How can I use loco with sqlite

### DISCUSSION THREAD

**Discussion by CharlieBytesX** [loco-rs/loco#441] [Q&A] [answered]
Hi, how  can I run loco with sqlite?

**Comment by kaplanelad** [comment, marked as answer]
Hello @charlieman2700, we've had a few discussions on this topic. Please refer to the following links for more details:
https://github.com/loco-rs/loco/discussions/349
https://github.com/loco-rs/loco/discussions/273
https://github.com/loco-rs/loco/discussions/235

**Reply by CharlieBytesX** [reply]
Thanks!
